Formula/libsquish.rb

```diff
@@ -19,8 +20,17 @@ class Libsquish < Formula
   depends_on "cmake" => :build
 
   def install
-    system "cmake", ".", *std_cmake_args
-    system "make", "install"
+    # Static and shared libraries have to be built using separate calls to cmake.
+    args = []
+    args << "-DBUILD_SQUISH_WITH_SSE2=OFF" if Hardware::CPU.arm?
+    system "cmake", "-S", ".", "-B", "build_static", *std_cmake_args, *args
+    system "cmake", "--build", "build_static"
+    system "cmake", "--install", "build_static"
```

Comment:
```suggestion
    lib.install "build_static/libsquish.a"
```
This will save the overhead of the extra `cmake` call. Though we'll need to check if `libsquish.a` will in fact be in `build_static` rather than a subdirectory thereof. (It's usually there though.)

Reply:
It looks like that works - `libsquish.a` gets installed as expected.